hack/debug.sh

```diff
@@ -61,6 +70,16 @@ log_system () {
 }
 
 do_master () {
+    services=($aos_master_service $aos_master_controllers_service $aos_master_api_service)
+    start_args=("master" "master controllers" "master api")
+    config_files=()
+    for i in "${!services[@]}"; do
+        config_files[$i]=$(get_config_path_from_service "${start_args[$i]}" "${services[$i]}")
+        if [ -z "${config_files[$i]}" ]; then
+            config_files[$i]="zzzz"
```

Comment:
maybe it would be easier to split a `log_service` function out of log_system and then just call that once for each service (below, after calling `log_system`) rather than trying to deal with bash arrays